conf/db/migration/metrics_portal_ddl/V18__job_timeout.sql

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2019 Dropbox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE portal.reports ADD COLUMN timeout_nanos BIGINT;
```

Comment:
Is this how we've done durations? I thought they were mostly converted to doubles.

Reply:
I only see three other durations in MPortal's tables: [`period_in_seconds INTEGER`](https://github.com/ArpNetworking/metrics-portal/blob/588f8616321c7043d2cf48105b223cc851f9c320/conf/db/migration/metrics_portal_ddl/V3__create_alerts_and_expressions_table.sql#L28), [`freshness_threshold_in_seconds INTEGER`](https://github.com/ArpNetworking/metrics-portal/blob/588f8616321c7043d2cf48105b223cc851f9c320/conf/db/migration/metrics_portal_ddl/V3__create_alerts_and_expressions_table.sql#L45), and [`offset_nanos BIGINT`](https://github.com/ArpNetworking/metrics-portal/blob/588f8616321c7043d2cf48105b223cc851f9c320/conf/db/migration/metrics_portal_ddl/V17__report_periodic_schedule_offset_to_double.sql#L17).